src/Microsoft.Framework.PackageManager/Commands/WrapCommand.cs

```diff
@@ -47,7 +47,7 @@ public bool ExecuteCommand()
             var extension = Path.GetExtension(InputFilePath);
             if (string.Equals(extension, ".csproj", StringComparison.OrdinalIgnoreCase))
             {
-                if (PlatformHelper.IsMono)
+                if (!RuntimeEnvironmentHelper.IsWindows)
```

Comment:
Yeah, previously without the utils added by this PR, we have no way to tell the OS we are running on. So we assumed `IsMono == non-Windows`. Now we are fixing this.